pandas/tests/test_nanops.py

```diff
@@ -355,22 +310,31 @@ def test_nanmean_overflow(self):
             assert result == np_result
             assert result.dtype == np.float64
 
-    def test_returned_dtype(self):
-
-        dtypes = [np.int16, np.int32, np.int64, np.float32, np.float64]
-        if hasattr(np, "float128"):
-            dtypes.append(np.float128)
+    @pytest.mark.parametrize(
+        "dtype",
+        [
+            np.int16,
```

Comment:
Not here and no action required but maybe want to expand to cover unsigned and 8 bit versions. I've seen that in a few other PRs so should become a fixture if not already

Reply:
Good idea

Reply:
we also have our fixtures for dtypes (not blocking this PR)